src/main/kotlin/uk/gov/justice/digital/hmpps/deliusapi/service/throughcare/RecallService.kt

```diff
@@ -190,7 +190,7 @@ class RecallService(
 
       var contactNotes = """
         Transfer Reason: ${allocationReason.description}
-        Transfer Date: ${previousPOM.allocationDate}
+        Transfer Date: ${previousPOM.allocationDate.toLocalDate()}
```

Comment:
The ticket mentions the date should be formatted as `dd/MM/yyyy`, whereas I think this will just chop the time off and print `yyyy-MM-dd`.

Might be better to use a formatter?  Also it'd be good to add a small unit test to check the notes come out correctly.
```suggestion
        Transfer Date: ${previousPOM.allocationDate.format(DateTimeFormatter.ofPattern("dd/MM/yyyy"))
}
```